.circleci/config.yml

```diff
@@ -16,6 +16,7 @@ workflows:
             sbt "scalafmt::test;
                  test:scalafmt::test;
                  sbt:scalafmt::test;
+                 universal:stage;
```

Comment:
What is this doing?

Reply:
This is creating the Java bundle used in the `dev` docker image.
Since I now merged the `Dockerfile`s in one, we need to do this first.
It is already documented here: https://github.com/codacy/codacy-swiftlint/pull/30/files#diff-04c6e90faac2675aa89e2176d2eec7d8R31

Reply:
Ok, just a couple of questions then
1. Then why is this needed in the CI if it's only for dev purposes
2. And are we building dev every time we do a docker build? Even if we want to test the production version?

Reply:
> Then why is this needed in the CI if it's only for dev purposes

Dockerfiles are "linear" in the sense that you can't fork and skip a step. So even if you don't use the `dev` stage you need to build it.

> And are we building dev every time we do a docker build? Even if we want to test the production version?

Yes, we are. This has, however, a good side effect. It allows us to test if the `dev` docker builds correctly.